packages/types/src/appearance.ts

```diff
@@ -94,8 +94,6 @@ export type ProfileSectionId =
   | 'organizationDomains';
 export type ProfilePageId = 'account' | 'security' | 'organizationSettings' | 'organizationMembers';
 
-export type NavbarItemId = 'account' | 'security' | 'members' | 'settings';
-
```

Comment:
If we are interested in keeping this we could do 

```tsx
export type NavbarItemId = 'account' | 'security' | 'members' | 'settings' | (string & {});

```

Reply:
Does it make sense to keep this?